assets/global-classes.less

```diff
@@ -228,6 +228,15 @@ textarea {
   margin: .5em 0 .5em 0;
 }
 
+label > .input-primary {

```

Comment:
Take this out of global-classes since I think it's going to have some unintended side effects. We're going to have to rework the global styling rules later anyways

Reply:
Should I put it in the scoped styling of each signup form component/view?

Reply:
Yeah that works

Reply:
Or if you want to verify that it works appropriately for every other form that uses a label then you can leave it here